src/algebra/order/chebyshev.lean

```diff
@@ -0,0 +1,123 @@
+/-
+Copyright (c) 2023 Mantas Bakšys, Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mantas Bakšys, Yaël Dillies
+-/
+import algebra.big_operators.order
+import algebra.order.rearrangement
+import group_theory.perm.cycle.basic
+
+/-!
+# Chebyshev's sum inequality
+
+This file proves the Chebyshev sum inequality.
+
+Chebyshev's inequality states `(∑ i in s, f i) * ∑ i in s, g i ≤ s.card * ∑ i in s, f i * g i` when
```

Comment:
```suggestion
Chebyshev's inequality states `(∑ i in s, f i) * (∑ i in s, g i) ≤ s.card * ∑ i in s, f i * g i` when
```
It's strange to have braces around the first sum and not the second sum. I know this is what Lean needs to understand, but docstrings are written for human readers. Even below in Lean statements, I think it would be better to have more symmetry like that.